packages/teleterm/src/ui/QuickInput/useQuickInput.ts

```diff
@@ -36,7 +36,7 @@ export default function useQuickInput() {
   const [activeSuggestion, setActiveSuggestion] = React.useState(0);
   const autocompleteResult = React.useMemo(
     () => quickInputService.getAutocompleteResult(inputValue),
-    [inputValue]
+    [inputValue, workspacesService.getActiveWorkspace()?.localClusterUri]
```

Comment:
Let's just add a comment here explaining why it's in the dependency list to make sure we're highlighting a potential code smell.